os/include/tinyara/lcd/ili9341.h

```diff
@@ -1,6 +1,6 @@
 /****************************************************************************
  *
- * Copyright 2018 Samsung Electronics All Rights Reserved.
+ * Copyright 2019 Samsung Electronics All Rights Reserved.
```

Comment:
why need change year here, is this newly add file?

Reply:
fixed, thanks